airbyte-ci/connectors/metadata_service/lib/metadata_service/gcs_upload.py

```diff
@@ -122,22 +122,22 @@ def _icon_upload(metadata: ConnectorMetadataDefinitionV0, bucket: storage.bucket
     return upload_file_if_changed(local_icon_path, bucket, latest_icon_path)
 
 
-def create_prerelease_metadata_file(metadata_file_path: Path, prerelease_tag: str) -> Path:
-    metadata, error = validate_and_load(metadata_file_path, [])
+def create_prerelease_metadata_file(metadata_file_path: Path, prerelease: str) -> Path:
+    metadata, error = validate_and_load(metadata_file_path, [], prerelease=prerelease)
```

Comment:
I find this naming to increase confusion - it's not clear to me upon first look elsewhere in the code if `prerelease` is going to contain a boolean or a tag. (I actaully thought in the ValidatorContext it would be a bool for example)